tests/core/test_externref.s

```diff
@@ -1,5 +1,6 @@
 # Define a global of type `externref`
 
+.section .data.my_global,"",@
```

Comment:
maybe add comments here that link to the llvm issue?

Reply:
The section directive is actually needed here.. that bug is that llvm crashes rather than given a nice error.

wasm globals should never be allowed in the .text section.   If we build without `-g` it seems that this is overlooked and seems to work.. but it shouldn't work.  

So I don't think we need to reference the LLVM issue here (since there will be nothing to change once its fixed).  But I have created an issue here: https://github.com/llvm/llvm-project/issues/56270

Reply:
Ah, I see, thanks!